projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -693,6 +693,21 @@
         width: 100%;
     }
 
+    @for $i from 1 through 10 {
+        %grid__group-indentation-level-#{$i} {
+            min-width: #{$i*30}px;
+            flex-basis: #{$i*30}px;
+
+            .igx-button--icon {
+                min-width: 30px;
+                flex-basis: 30px;
+            }
+        }
```

Comment:
The smallest grouping-indicator should be min-width:60px, flex-basis: 60px
![](https://user-images.githubusercontent.com/13866657/40716643-55414a3c-6412-11e8-8d4f-7162fafeb034.png)

Reply:
Respectively the smallest group-indentation should be 60px

Reply:
I will test and do that change

Reply:
Ok. Thank you  :+1: